unattended.sh

```diff
@@ -110,19 +110,34 @@ if [ -z "$timezone" ] ; then
   timezone=UTC
 fi
 echo "date.timezone = $timezone" > /etc/opt/rh/rh-php71/php.d/10-centreon.ini
-if [ "x$has_systemd" '=' x1 ] ; then
-  systemctl restart rh-php71-php-fpm
-fi
 print_step_end "OK, timezone set to $timezone"
 
+#
+# FIREWALL
+#
+
+print_step_begin "Firewall configuration"
+command -v firewall-cmd > /dev/null 2>&1
+if [ "x$?" '=' x0 ] ; then
+  for svc in http mysql snmp snmptrap ; do
```

Comment:
I would not recommend leaving mysql port service open

Reply:
Any opinion on this @lpinsivy ?

Reply:
OK I removed _mysql_ from the service list.